### PR TITLE
frontend: provide statistics on how many users submitted at least one build in past month

### PR DESCRIPTION
**Pull request by xsuchy**
Fixes: #3315

### CODE REVIEW

File: frontend/coprs_frontend/coprs/logic/builds_logic.py [hunk 1]
```diff
@@ -129,6 +129,18 @@ def get_running_tasks_by_time(cls, start, end):
...
         return result
 
+    @classmethod
+    def get_users_by_time(cls, start, end):
+        """ Returns number of user that submitted at least one build in
+            the period.
+        """
+        result = db.session.query(models.Build.user_id)\
+            .join(models.BuildChroot)\
+            .filter(models.BuildChroot.ended_on > start)\
```
Comment (praiskup) [inline frontend/coprs_frontend/coprs/logic/builds_logic.py:139]: This filter seems redundant.  To check that the user is active, it is enough to monitor the "started_on" field.

Comment (xsuchy) [reply frontend/coprs_frontend/coprs/logic/builds_logic.py:139]: True for the params we are passing. But not true for any time period.
And I basically steal this from the fuction above.

**Comment by xsuchy** [comment]
The failing lint is on file that I did not touch. I am going to ignore it.

**Comment by praiskup** [comment]
> The failing lint is on file that I did not touch. I am going to ignore it.

Yes, please ignore.  These files have been modified recently (the warnings disappeared) in the `main` branch, and since the linter doesn't rebase - your latest commit still contains those  (the diff is done against the old and new code).

**Comment by praiskup** [comment]
I believe I finally [fixed the linter](https://github.com/fedora-copr/vcs-diff-lint-action/blob/75074d785212a9276d8f5219783b7d051e8d323a/container/cmd#L91-L93), seems green now.

**Review by praiskup — APPROVED** [review]

**Pull request merged**